test/system/030-run.bats

```diff
@@ -901,4 +901,22 @@ $IMAGE--c_ok" \
     run_podman rm $ctr_name
 }
 
+@test "podman run --privileged as root with systemd will not mount /dev/tty" {
+    skip_if_rootless "this test only makes sense as root"
+
+    ctr_name="container-$(random_string 5)"
+    run_podman run --rm -d --privileged --systemd=always --name "$ctr_name" "$IMAGE" /home/podman/pause
+
+    TTYs=$(ls /dev/tty*|sed '/^\/dev\/tty$/d')
+
+    if [[ $TTYs = "" ]]; then
+        die "Did not find any /dev/ttyN devices on local host"
+    else
```

Comment:
It's a little bit confusing to use `else` after `die`, since by definition `die` terminates execution... but not worth re-pushing for, especially given our flaky CI today. I'll fix it in one of my periodic cleanup PRs.

LGTM, but I'll let other more systemd-savvy team members give final approval. Thanks again!